docs/guides/modules/utils.rst

```diff
@@ -71,7 +71,19 @@ To use it, import the decorator and apply it to the class for a form:
        def __init__(self, **properties):
            self.init_components(**properties)
 
-Now, the form has an ``item`` property which behaves like a dictionary. Whenever a value of that dictionary changes, the form's ``refresh_data_bindings`` method will be called.
+Now, the form has an ``item`` property which behaves like a dictionary. Whenever a value of that dictionary changes, the form's ``refresh_data_bindings`` method will be called. 
+
+Note: The ``item`` property will no longer reference the same object after an auto-refresh. Rather, in the following example, it is as though auto-refresh adds the ``item = dict(item)`` line:
```

Comment:
Can we remove `after and auto-refresh` from this sentence? This happens when the form is instantiated rather than when a refresh occurs.